### ISSUE TITLE
Improve error message and bridge UI when sequence in shipyard has no tasks

### ISSUE THREAD

**Issue by grabnerandi** [keptn/keptn#6790]
**Is your feature request related to a problem? Please describe.**
We ran into a problem and spent several hours of troubleshooting on why an evaluation sequence didnt execute any tasks. Turned out that the shipyard file didnt have any tasks specified for that evaluation sequence. 
```
kind: shipyard
spec:
  stages:
  - name: testing
    sequences:
    - name: evaluation
```

The reason we ended up in the situation of not having any tasks was because of the wrong assumption that Keptn will default to a sequence with the evaluation task in case the task list is empty. This assumption was wrong as Keptn only defaults to an evaluation sequence if the evaluation sequence itself is not defined at all.

The shipyard controller immediately sends an evaluation.finished with a result of pass. But - because there are no tasks executed in the sequence the keptn bridge shows the sequence as still executing. This is why we ended up troubleshooting a long time as we thought the sequence is still doing something but nothing ever happened

![image](https://user-images.githubusercontent.com/675886/152801380-18c8d8c2-838a-421b-9cf2-3509c9966154.png)

**Describe the solution you'd like**
I suggest two things
a) the SEQUENCENAME.triggered event sent by the shipyard-controller should not use pass as a result but fail with the error message that the sequence has no task
b) the bridge needs to not show this sequence is still processing

**Comment by johannes-b** [comment]
In the docs, the explanation of the behavior is mentioned here - at "Why I do not need to specify a task sequence for quality gates in this stage?": https://keptn.sh/docs/0.12.x/quality_gates/get_started/#definition-of-project-stage-and-service

We should highlight it more prominently to avoid this issue again.

**Comment by ermin-muratovic** [comment]
This is not a bug in the Bridge, since the GET `/sequence` endpoint return state `started` for this sequence, which should be `finished` in this case.
![image](https://user-images.githubusercontent.com/6098219/155957497-168f5814-c987-42f1-b569-4267fc2afee3.png)

**Comment by odubajDT** [comment]
Hi,

may I please ask you for the exact steps to reproduce the problem?

I was unable to reproduce it, my steps were:

1. Creating a project `keptn create project test --shipyard=./short-shipyard.yaml` with shipyard
`apiVersion: "spec.keptn.sh/0.2.3"
kind: "Shipyard"
spec:
  stages:
  - name: testing
    sequences:
    - name: evaluation`
2. creating service `keptn create service helloservice --project=test`
3. triggering evaluation `keptn trigger evaluation --project=test --stage=testing --service=helloservice --start=2022-03-26T10:05:53.931Z --end=2022-03-26T10:10:53.931Z`

Result:
/home/odubaj/Pictures/Screenshot from 2022-02-28 16-18-48.png

Thank you in advance!

**Comment by odubajDT** [comment]
From my investigation this problem was fixed with this commit 

https://github.com/keptn/keptn/commit/bdcd95e5c5857a7c5ef0abff49524cafdf2a8b86

and is fixed in version 0.13.0, see release notes
https://github.com/keptn/keptn/releases/tag/0.13.0

it's not fixed in 0.12.2
https://github.com/keptn/keptn/releases/tag/0.12.2

**Comment by odubajDT** [comment]
Closing this issue, as we do not plan to backport the fix to version 0.12.2
